src/Chapter3.hs

```diff
@@ -480,6 +526,45 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city in total.
 -}
 
+data Wall = MkWall
+
+data Castle = ACastle String Wall | NoCastle
```

Comment:
With this data type representation, you will always have walls inside the castle. But the task says that a castle might not have a wall at all.